tests/expected_cmdline_output_files/basic_connectivity_expl_output.txt

```diff
@@ -0,0 +1,14775 @@
+final fw rules for query: , config: testcase8-networkpolicy1.yaml:
+src: 0.0.0.0/0 dst_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
+src_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] src_pods: [*] dst_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] dst_pods: [*] conn: All connections
+<entry dst="0.0.0.0/0" id="1" src="0.0.0.0/0">
```

Comment:
add a newline space before the explanations section start + a text header

Reply:
I think it might be more useful to print the explanations in some textual format rather than xml (maybe consider this as another follow up issue to consider), especially when appended to the connectivity textual output

Reply:
After discussion, the 'ALL' option will be used for internal usage. The 2-nodes scenario already has a newline separation.